draft-schedulers.mkd

```diff
@@ -411,8 +409,8 @@ receive-window, this scheduler deploys a fast reinjection mechanism to quickly
 recover from the head-of-line blocking.
 
 At each round, the scheduler iterates over the list of paths that are eligible
-for transmission. To decide whether or not a path a few conditions need to be
-satisfied:
+for transmission. To decide whether or not a path is active, a few conditions
```

Comment:
I am not sure about the word "active" which sounds here like "is currently used". I guess we want to say more something like: "can be used". Or "is eligible". No?

Reply:
yep, is eligible is better